libnmstate/schema.py

```diff
@@ -210,3 +212,80 @@ class OVSBridge(object):
 class OVSBridgePortType(object):
     SYSTEM = 'system'
     INTERNAL = 'internal'
+
+
+class Team(object):
+    TYPE = InterfaceType.TEAM
+    CONFIG_SUBTREE = 'team'
+
+    SLAVES = 'slaves'
+    CONFIG = 'config'
+
+
+class TeamConfig(object):
+    CONFIG = Team.CONFIG
+
+    DEVICE = 'device'
+    DEBUG_LEVEL = 'debug_level'
```

Comment:
Please use `-` instead of `_`.

Reply:
We are using `_` for all the other constants. Why should we use `-` for team interface constants?

Reply:
Which API key has an underscore?

Reply:
Not the API key string but the constant name. If we replace `_` by `-` in all API key strings, then we would need to replace `-` by `_` again before send the plain text JSON to NetworkManager. Is that okay with you?

Reply:
Sorry, I do not understand what you are referring to. A constant is python is a variable name, which cannot use the `-` character. The original comment was on the value of the variable/constant, which is the string used on the json-schema as a key.

Reply:
Yes, I was meaning the value of the variable. The teamd JSON API uses underscore for all the keys. e.g `debug_level`. In order to set the team configuration we need to send the plain text JSON to NetworkManager so, we would need to replace all `-` by `_`.  This is why I've decided to use underscore in first place. All teamd json keys are listed here `man  5 teamd.conf`.

I hope it's more clear now. Thanks!

Reply:
Ok, now I understand, thank you for the clarification.
The Nmstate API is not trying to mimic NM API as a general rule. We may mimic or follow naming and models from NM if it fits the needs.

In this case, we use all over the `-` alternative, which is more user friendly than the `_` option (which is developer friendly). We should be consistent here.
In addition, the exposed API should fit the declarative nature, so please do not assume every parameter deserves exposing.

Reply:
That seems fine to me. Thanks for explaining

Reply:
Removed debug stuff and added more tests in order to cover all the configuration parameters.